nat-lab/tests/utils/connection/ssh_connection.py

```diff
@@ -32,9 +31,5 @@ async def get_ip_address(self) -> tuple[str, str]:
         ip = self._connection._host  # pylint: disable=protected-access
         return (ip, ip)
 
-    async def mapped_ports(self) -> tuple[str, str]:
```

Comment:
This can be removed since it's now the same as the default implementation in `connection.py`